### PR TITLE
Worker task template

### LINKED ISSUE #142: Add template (model) for workers tasks

### PR DESCRIPTION
**Pull request by lurak**

### CODE REVIEW

**Review by daniiloleshchuk — APPROVED** [review]

File: stock_dashboard_api/utils/worker_task.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import json
+
+from constants import FETCH_HISTORICAL_DATA_TASK,\
+    FETCH_DATA_FOR_PERIOD_TASK, FETCH_NEW_STOCK_TASK
```
Comment (lhalam) [inline stock_dashboard_api/utils/worker_task.py]:
```
from constants import (FETCH_HISTORICAL_DATA_TASK,   
                       FETCH_DATA_FOR_PERIOD_TASK, 
                       FETCH_NEW_STOCK_TASK)
```

File: stock_dashboard_api/utils/worker_task.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    TO_INDEX = 2
+
+    def __init__(self, task_id):
+        """
+        Initialization of task id
+        :param task_id: number of task
+        """
+        self.task_id = task_id
+
+    def form_task(self, *params):
```
Comment (lhalam) [inline stock_dashboard_api/utils/worker_task.py]: ??? *

File: stock_dashboard_api/utils/worker_task.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    FETCH_DATA_FOR_PERIOD_TASK, FETCH_NEW_STOCK_TASK
+
+
+class Task:
+    """
+    Class for representation of different tasks
+    """
+    STOCK_NAME_INDEX = 0
+    FROM_INDEX = 1
+    TO_INDEX = 2
```
Comment (lhalam) [inline stock_dashboard_api/utils/worker_task.py]: remove

File: stock_dashboard_api/utils/worker_task.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+    def __init__(self, task_id):
```
Comment (lhalam) [inline stock_dashboard_api/utils/worker_task.py]: add param `stock_name` `from` `to`

File: stock_dashboard_api/utils/worker_task.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+        elif self.task_id == FETCH_NEW_STOCK_TASK:
+            body = json.dumps({"task_id": FETCH_NEW_STOCK_TASK,
+                               "stock_name": params[Task.STOCK_NAME_INDEX]})
+            return body
+        elif self.task_id == FETCH_HISTORICAL_DATA_TASK:
+            body = json.dumps({"task_id": FETCH_HISTORICAL_DATA_TASK,
+                               "stock_name": params[Task.STOCK_NAME_INDEX],
+                               "from": params[Task.FROM_INDEX],
+                               "to": params[Task.TO_INDEX]})
+            return body
```
Comment (lhalam) [inline stock_dashboard_api/utils/worker_task.py]: decomposition metod

**Review by ghost — APPROVED** [review]

**Review by OstapOmelchuk — APPROVED** [review]

**Review by StepanTchynetskyi — APPROVED** [review]

**Pull request merged**